src/modules/peek/Peek.UI/Views/TitleBar.xaml

```diff
@@ -95,39 +102,42 @@
             <VisualStateGroup x:Name="AdaptiveWidth">
                 <VisualState x:Name="MaximumLayout">
                     <VisualState.StateTriggers>
-                        <AdaptiveTrigger MinWindowWidth="720" />
+                        <AdaptiveTrigger MinWindowWidth="560" />
                     </VisualState.StateTriggers>
                     <VisualState.Setters>
-                        <Setter Target="AppTitle_FileName.MaxWidth" Value="560" />
+                        <Setter Target="AppTitle_FileCount.Visibility" Value="Visible" />
```

Comment:
This should not be set as it is normally bound to IsMultiSelection. This means it will be shown when we do not want.

Reply:
Kk! Good to know. Will remove it and change the width boundaries accordingly